README.md

```diff
@@ -1,19 +1,17 @@
-<!-- markdownlint-disable-file MD033 - inline HTML is allowed here -->
-<!-- markdownlint-disable-next-line MD041 - okay to NOT start file with top-level heading -->
 <p align="center">
   <img height="100" src="https://phylum.io/logo/dark-bckg.svg">
 </p>
 
 ---
 
-# Command Line Utility for [Phylum.io](https://phylum.io)
-  
+# Introduction
+
 ![GitHub release (latest by date)](https://img.shields.io/github/v/release/phylum-dev/cli)
 [![MIT License](https://img.shields.io/apm/l/atomic-design-ui.svg?)](https://github.com/tterb/atomic-design-ui/blob/master/LICENSEs)
 ![Test Status](https://github.com/phylum-dev/cli/actions/workflows/test.yml/badge.svg?branch=master)
 [![README](https://img.shields.io/badge/docs-README-yellowgreen)](https://docs.phylum.io/docs/welcome)
 
-The command line interface (CLI) allows users to submit their project package dependencies to Phylum's API for analysis. Currently [pre-built binaries](https://github.com/phylum-dev/cli/releases) for Linux and macOS are available. For other platforms (e.g. Windows), binaries can easily be [built](https://github.com/phylum-dev/cli#building). Additional information about using the CLI tool is available in the [documentation](https://docs.phylum.io/docs/welcome).
+The command line interface (CLI) allows users to submit their project package dependencies to [Phylum's](https://phylum.io) API for analysis. Currently [pre-built binaries](https://github.com/phylum-dev/cli/releases) for Linux and MacOS are available. For other platforms (e.g., Windows), binaries can easily be [built](https://docs.phylum.io/docs/building).
```

Comment:
It may seem unnatural, but `macOS` (with a lowercase `m`) is the [official branding](https://web.archive.org/web/20161111191151/https://www.businessinsider.com/wwdc-2016-os-x-becomes-macos-2016-6). Apple wanted it to fit in with `iOS` and friends.

Reply:
Fixed in both `README.md` and `quickstart.md`